src/command_modules/azure-cli-network/tests/test_network_commands.py

```diff
@@ -1122,22 +1111,63 @@ def test_network_watcher(self, resource_group, storage_account):
 
         self.cmd('network watcher test-connectivity -g {} --source-resource {} --dest-address www.microsoft.com --dest-port 80'.format(resource_group, vm))
 
-        # TODO: Re-enable once issue #3385 is resolved
-        # capture = 'capture1'
-        # location = 'westus'
-        # self.cmd('network watcher packet-capture create -g {} --vm {} -n {} --file-path capture/capture.cap'.format(resource_group, vm, capture))
-        # self.cmd('network watcher packet-capture show -l {} -n {}'.format(location, capture))
-        # self.cmd('network watcher packet-capture stop -l {} -n {}'.format(location, capture))
-        # self.cmd('network watcher packet-capture show-status -l {} -n {}'.format(location, capture))
-        # self.cmd('network watcher packet-capture list -l {}'.format(location, capture))
-        # self.cmd('network watcher packet-capture delete -l {} -n {}'.format(location, capture))
-        # self.cmd('network watcher packet-capture list -l {}'.format(location, capture))
-
         nsg = '{}NSG'.format(vm)
         self.cmd('network watcher flow-log configure -g {} --nsg {} --enabled --retention 5 --storage-account {}'.format(resource_group, nsg, storage_account))
         self.cmd('network watcher flow-log configure -g {} --nsg {} --retention 0'.format(resource_group, nsg))
         self.cmd('network watcher flow-log show -g {} --nsg {}'.format(resource_group, nsg))
 
 
+class NetworkWatcherPacketCaptureScenarioTest(ScenarioTest):
+    import mock
```

Comment:
`mock` is standard, especially for tests. Would consider just having it at the top of the file.